svc-api/handle/compositionservice.go

```diff
@@ -0,0 +1,523 @@
+//(C) Copyright [2022] American Megatrends International LLC
```

Comment:
Use multiline comments for license header. Change this in all the files

Reply:
we followed the same format of other odim files. For example : In svc-api/handle/chassis.go, single line comments are used.